_docs/setup/kubernetes/multicluster-install.md

```diff
@@ -14,9 +14,12 @@ Instructions for the installation of Istio multicluster.
 
 ## Prerequisites
 
-* Two or more Kubernetes clusters.
+* Two or more Kubernetes clusters with **1.7.3 or newer**.
 
-* The usage of an RFC1918 network, VPN, or other more advanced network techniques
+* The ability to deploy the [Istio control plane]({{home}}/docs/setup/kubernetes/quick-start.html)
+on **one** Kubernetes cluster.
+
+* The usage of an RFC1918 network, VPN, or alternative more advanced network techniques
```

Comment:
@gyliu513 This is where the pre-reqs are specified requiring pod IPs to be routable between each cluster.

Reply:
This is a very big limitation I think, I think that most of the clusters cannot make sure the pod ip is accessible from all clusters.

Reply:
Thanks @sdake , any github issue opened for this?